gaudi/README.md

```diff
@@ -0,0 +1,89 @@
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_nfd_instance_openshift.yaml
+```
+Verify NFD has labelled the node correctly:
+```
+oc get no -o json | jq '.items[].metadata.labels' | grep pci-1da3
+
+"feature.node.kubernetes.io/pci-1da3.present": "true",
+```
+NFD detects underlying Gaudi Accelerator using its PCI device class and the vendor.
+
+## Install HabanaAI Operator on Red Hat OpenShift
+### Installation via web console
+Follow the steps below to install HabanaAI Operator using OpenShift web console:
+1.	In the OpenShift web console, navigate to **Operator** -> **OperatorHub**.
+2.	Search for **HabanaAI Operator** in all items field -> Click **Install**.
+### Verify Installation via web console
+1.	Go to **Operator** -> **Installed Operators**.
+2.	Verify that the status of the operator is **Succeeded**.
+
+### Installation via command line interface (CLI)
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_install_operator.yaml
+```
+
+### Verify Installation via CLI
+Verify that the operator controller manager pod is up and running:
+```
+oc get pod -n habana-ai-operator
+
+NAME                                  READY   STATUS    RESTARTS   AGE
+controller-manager-6c8459d9cb-fqs8h   2/2     Running   0          25m
+```
+
+## Resources Provided by Habana Gaudi Device Plugin
+The resources provided are the user interface for customers to claim and consume the hardware features from the user pods. See below table for the details:
+
+| Feature | Resources | Description |
+| ------- | --------- | ----------- |
+| Habana Gaudi | `habana.ai/gaudi` | Number of Habana Gaudi Card resources ready to claim | 
+
+## Creating Habana AI Operator DeviceConfig Instance
+To create a Habana Gaudi device plugin CR, follow the steps below.
+
+### Create CR via web console
+1.	Go to **Operator** -> **Installed Operators**.
+2.	Open **HabanaAI Operator**.
+3.	Navigate to tab **Device Config**.
+4.	Click **Create DeviceConfig** -> set correct parameters -> Click **Create**. To set correct parameters please refer [Using RedHat OpenShift Console](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/Deploying_HabanaAI_Operator.html#id2).
+
+### Verify via web console
+1.	Verify CR by checking the status of **Workloads** -> **DaemonSet** -> **habana-ai-module-device-plugin-xxxxx**.
+2.	Now `DeviceConfig` is created.
+
+### Create CR via CLI
+Apply the CR yaml file:
+```
+oc apply -f  https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_device_config.yaml
+```
+
+### Verify via CLI
+Verify that the device plugin CR is ready:
+```
+oc get DeviceConfig -n habana-ai-operator --show-kind=true
+NAME                               AGE
+deviceconfig.habana.ai/habana-ai   2d8h
+```
```

Comment:
Can we add one more section to verify that the Provisioning is successful?

## Verifying Gaudi provisioning
After the DeviceConfig instance is created, It will take some time for the operator to download the Gaudi OOT driver source code and build it on-premise with the help of the KMM operator. The OOT driver module binaries will be loaded into the RHCOS kernel on each node with Gaudio Cards labelled by NFD. Then, the Gaudi device plugin can advertise the Gaudi resources listed in the table for the pods on OpenShit to use.
Use the command below to check the availability of Gaudi resources.
@chaitanya1731  please add the command to check the Gaudi resources, and move the table and related paragram here. 

@chaitanya1731 It is great if we can add command to check  nodeMetrics and runtime. 
Thanks,